packages/web-components/src/components/ic-data-table/ic-data-table.css

```diff
@@ -0,0 +1,222 @@
+:host {
+  position: relative;
+  display: block;
+  height: 100%;
+  max-height: 100%;
+}
+
+.table-container {
+  position: relative;
+  height: calc(100% - var(--ic-space-xxxs));
+}
+
+.table-row-container {
+  position: relative;
+  height: 100%;
+  overflow: auto;
+  transition: var(--ic-easing-transition-fast);
+  overflow-anchor: none;
+}
+
+:host([show-pagination="true"]) .table-row-container {
+  height: calc(100% - 3.5625rem);
+}
+
+table {
+  table-layout: auto;
+  border-spacing: 0;
+  width: 100%;
+}
+
+.table-row-container:focus {
+  outline: none;
+  box-shadow: var(--ic-border-focus);
+  border-radius: var(--ic-border-radius);
+  transition: var(--ic-easing-transition-fast);
+  z-index: 1;
+}
+
+:host([embedded="true"]) .table-container {
+  border: solid var(--ic-architectural-200) var(--ic-space-1px);
```

Comment:
Is it worth creating a new border variable for this? I think it's used a few times in navigation menu etc.